operating/on_sky.rst

```diff
@@ -0,0 +1,113 @@
+   * In the web interface, move the acquisition mirror to the ``Out`` position. Now, increase ``EM gain`` under ``camwfsctrl`` (for fainter targets an EM gain of ``600`` is sufficient). This will put more light on camwfs. 
+   * Now, look at ``camwfs``. Reduce the step size under ``Move Telescope`` and finely adjust the telescope pointing until the pupils on ``camwfs`` appear evenly illuminated.
+
+6. Closing the Loop
+
+   * In the ``OffloadingCtrl`` GUI, drag the slider to ``On`` and set the gain to ``0.1``. |Image5|
+   * In the ``holoop`` GUI, click ``Zero All`` to remove any existing loop gain settings.
+   * Raise the global gain to ``1.0``.
+   * Drag the slider from the ``Open`` to the ``Closed`` position. 
+   * Raise tip and tilt to gains of roughly ``0.1`` - ``0.2``. 
+   * In the ``OffloadingCtrl`` GUI under ``Focus``, press ``Dump``.
+   * Monitor the DM displays to ensure that the DMs are not heavily saturating (the DM will turn red when it saturates). 
+   * If the loop reopens, press ``Zero`` in the ``holoop`` GUI and ``Zero`` in ``OffloadingCtrl`` before attempting to close the loop again.
+
+On-Sky Alignment
+----------------
+
+Pupil Alignment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+1. Tweeter
+
+   * See the `F-test guide <./alignment>`__ for documentation regarding alignment of the tweeter. 
+
+2. NCPC
+
+   * In the ``Pupil Alignment``  GUI under ``NCPC``, press ``Set Test``. 
+   * In ``camsci1Cntrl`` set ``stagesci1`` to the ``J test`` focus position and ``EM Gain`` to ``25``.  You should see a test pattern appear on the pupil. 
+   * In the ``Pupil Alignment`` GUI below ``TTM Peri``, use the arrow keys to align the pupil on the NCPC DM.
+   * Once aligned, set ``stagesci1`` back to FPM. 
+
+Coronagraph Alignment 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Coming soon...
+
+Bump Mask Alignment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. In the web interface, set ``fwpupil`` to ``bump-mask``. 
+#. Now use the ``Pupil Plane`` buttons in the ``coronAlignGUI`` to align the bump mask on the pupil. Set the step size to ``100`` and press the down arrow 3 times. This is typically enough to align the bump mask. However, if the bump mask still appears slightly misaligned, reduce the step size to ``5`` and use the left/right arrow keys for fine adjustments.
+
+Closing the Pupil Alignment Loop
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. In the ``Pupil Alignment`` GUI, use the keypad in the lower-righthand corner to drive the x and y avg values to 0. After each translation, wait a few seconds for the readings to stabilize. Also note that the x-axis is much more sensitive than the y-axis. When moving x-axis avg to 0, use step sizes of about 0.1. When moving y-axis to 0, higher step sizes can be used.

```

Comment:
the idea is correct.  It's really something like "when x is within 0.2, steps sizes of 0.05 should be used.  The y-axis steps size can remain 0.1"